src/config/oAuth2.ts

```diff
@@ -0,0 +1,10 @@
+// base config
+const oAuth2Config = {
+    issuer: 'https://hydra.livesalepro.com',
+    clientId: 'lsp-encoder-app',
+    redirectUrl: 'lspencoderapp:/oauth',
+    scopes: ['openid','offline'],
+    clientSecret: '7oOKfHDCvtMc9txjlzpYxHhVms1bWYjT'
```

Comment:
@salmanmaredia Do not keep secret in code

Reply:
Should I place it in .env file or something? It wouldn't be secure anyways. Ideally we shouldn't need this secret on the client side. It should be configured on server somewhere.